js/app/Bounds.js

```diff
@@ -189,6 +186,76 @@ define(["app/Class"], function(Class) {
       return res;
     },
 
+    getBounds: function () { return this; },
+
+    update: function () {
+      /* Usages:
+
+         update("left,bottom,right,top")
+         update([left,bottom,right,top]);
+         update(obj, obj, ...);
+       */
+
+      var self = this;
+
+      var updateOne = function (obj) {
+        var left = undefined;
+        var bottom = undefined;
+        var right = undefined;
+        var top = undefined;
+
+
+        if (obj.length !== undefined) {
+          if (typeof(obj) == "string") {
+            obj = obj.split(",");
+          }
+          self.left = parseFloat(obj[0]);
+          self.bottom = parseFloat(obj[1]);
+          self.right = parseFloat(obj[2]);
+          self.top = parseFloat(obj[3]);
+        } else {
+          if (obj.getBounds != undefined) {
```

Comment:
In practice this should not make any difference, but I guess the compiler's being nit-picky as usual ;)